### PR TITLE
Make highlighted inline text bold only

### PR DESCRIPTION
**Pull request by tuantinghuang**
Relate #346 

Screenshot 2023-07-28 at 3 45 56 PM

### CODE REVIEW

**Comment by tuantinghuang** [comment]
The` <em> `tag by default make texts italic. The root solution might be going into Notion and make all the bold italic font bold only, so that the html wouldn't be consisted of `<strong><em>Text<em><strong>`, but instead `<strong> Text <strong>`. This pull request is changing the styling from the CSS side, instead of Notion.

Screenshot 2023-07-28 at 3 49 10 PM

**Comment by shiffman** [comment]
> The`\<em\>`tag by default make texts italic. The root solution might be going into Notion and make all the bold italic font bold only, so that the html wouldn't be consisted of `\<strong\>\<em\>Text\<em\>\<strong\>`, but instead `\<strong\> Text \<strong\>`. This pull request is changing the styling from the CSS side, instead of Notion.
> 
> Screenshot 2023-07-28 at 3 49 10 PM

Ah, yes, I think we should just fix the text in Notion as there's no reason to remove the possibility of having bold+italic text if we want it in the future. If you agree @tuantinghuang we can close this pull request and make the fixes in Notion itself.

**Comment by shiffman** [comment]
Noting also from #351 that some of the bold text has a "font-weight" (is this the right CSS property?) that is too high. Let's look at all "bold" text in the PDF and try to make it consistent?

Screen Shot 2023-08-02 at 7 55 25 AM

**Comment by tuantinghuang** [comment]
I went through all 10 chapters in Notion and changed most bold italic text to just bold. There are a couple of exceptions - 
- Some number/bullet lists are left as is. I saw three versions (bold, italic, bold italic) throughout the book. We might need to decide on one styling!

- The bold italic comments in code blocks are left as is, as I'm unsure what kind of treatment they should have. 
Screenshot 2023-08-04 at 2 15 03 PM

**Review by tuantinghuang — COMMENTED** [review]
After adjusting in Notion and removing the styling for `<em>` tag, the double bolded issue as mentioned in the earlier comment is also solved!

This is before & after:
Before
Screenshot 2023-08-04 at 2 33 06 PM

After
Screenshot 2023-08-04 at 2 33 37 PM

and fixed Chapter 2 -
Screenshot 2023-08-04 at 2 34 12 PM

**Review by jasongao97 — CHANGES_REQUESTED** [review]
Looking great! One tiny thing, could you please add a space between the selector and the `{`. It's a [common standard](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation) and also what we use in this project.

File: magicbook/stylesheets/components/typography.scss [hunk 1]
```diff
@@ -37,6 +37,10 @@ p {
   font-weight: $font-book;
 }
 
+strong{
```
Comment (shiffman) [inline magicbook/stylesheets/components/typography.scss]:
```suggestion
strong {
```

**Comment by shiffman** [comment]
@jasongao97 I added the space, if it looks good to you, please merge!

**Comment by shiffman** [comment]
And thank you @tuantinghuang!

**Comment by jasongao97** [comment]
Thank you so much!

**Pull request merged**